packages/react-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "publicodes-react",
-	"version": "1.0.0-beta.26",
+	"version": "1.0.0-beta.27",
 	"description": "UI to explore publicodes computations",
```

Comment:
Il faut pas ajouter aussi une ligne : 
```json
"type": "module"
``` 
pour signifier que le package export des ESModule ?

Reply:
Si effectivement

Je suis pas 100% content d'être obligé de changer le format, mais c'est la seule chose que j'ai réussi à faire fonctionner en local (sachant que l'affichage de la documentation fonctionne en dev mais pas en prod, donc le cycle d'itération pour tester est assez long...)